src/views/common/TimeScrollView/TimeScrollViewInteractions/TimeScrollViewEventHandlers.ts

```diff
@@ -0,0 +1,113 @@
+import { useTimeFocus, useTimeRange } from 'contexts/RecordingSelectionContext';
```

Comment:
This file pulls out the interaction/control logic that interrupted the flow of the TimeScrollView component itself. Instead here we can focus on the interactions as pure functions, and also define a hook that sets up callbacks for each that take an appropriate mouse event. Hopefully this will make it easier to see what the interactions themselves individually do, as well as putting all the dependencies in one small place.